gobblin-config-management/src/main/java/gobblin/config/utils/PathUtils.java

```diff
@@ -0,0 +1,87 @@
+package gobblin.config.utils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.apache.hadoop.fs.Path;
```

Comment:
I am not sure we should depend on this. This will make the client implementation dependent on Hadoop which it should not be.